lib/kafo/exit_handler.rb

```diff
@@ -34,7 +34,7 @@ def exit(code, &block)
     end
 
     def translate_exit_code(code)
-      return code if code.is_a?(Fixnum)
+      return code if code.is_a?(Integer) || code.is_a?(Fixnum)
```

Comment:
will there be cases, where Integer doesn't suffice?

Reply:
Good question. I'm not that familiar with Ruby so I figured this was safer but it's supposed to be a base class so is just Integer sufficient?

Reply:
I would say yes.